bot/bot.go

```diff
@@ -36,10 +37,10 @@ func (b *AIBot) Go() error {
 	return nil
 }
 
-func NewAIBot(botCtx context.Context, aiClient *gpt.Client, discordSession *discordgo.Session, storage *storage.Storage, imageStorage *storage.ImageStorage, logger *zap.Logger) *AIBot {
+func NewAIBot(botCtx context.Context, aiClient *gpt.Client, discordSession *discordgo.Session, storage *storage.Storage, imageStorage *storage.ImageStorage) *AIBot {
 	promptBytes, err := os.ReadFile("prompts/danbo.json")
 	if err != nil {
-		logger.Panic("Failed to read initial prompt", zap.Error(err))
+		log.Panic("Failed to read initial prompt", err)
```

Comment:
`slog` doesn't have an equivalent to `Panic` or `Fatal`, but because it's been configured as the default logger if we call these functions through the `log` package they'll still emit structured logs and then terminate execution